src/tests/Microsoft.Diagnostics.NETCore.Client/GetProcessInfoTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Diagnostics.NETCore.Client
+{
+    public class GetProcessInfoTests
+    {
+        private readonly ITestOutputHelper output;
+
+        public GetProcessInfoTests(ITestOutputHelper outputHelper)
+        {
+            output = outputHelper;
+        }
+
+        [Fact]
+        public void BasicProcessInfoTest()
+        {
+            using TestRunner runner = new TestRunner(CommonHelper.GetTraceePathWithArgs(targetFramework: "net5.0"), output);
+            runner.Start();
+
+            try
+            {
+                DiagnosticsClient client = new DiagnosticsClient(runner.Pid);
+
+                ProcessInfo processInfo = client.GetProcessInfo();
+
+                Assert.NotNull(processInfo);
+                Assert.Equal(runner.Pid, (int)processInfo.ProcessId);
+                Assert.NotNull(processInfo.CommandLine);
+                Assert.NotNull(processInfo.OperatingSystem);
+                Assert.NotNull(processInfo.ProcessArchitecture);
+                //Assert.Equal("Tracee", processInfo.ManagedEntrypointAssemblyName);
+                //Version clrVersion = ParseVersionRemoveLabel(processInfo.ClrProductVersionString);
+                //Assert.True(clrVersion >= new Version(6, 0, 0));
```

Comment:
You can test the exact value using `System.Reflection.Assembly.GetExecutingAssembly().GetName().Name`.

Reply:
The entry point name is that of the test application, not of the test host process or the test assembly. So this needs to be hard corded to the test application entrypoint name.

Reply:
Ah, good point. Could you get it from the ProcessModule name of the testee process? Theoretically that should be the same value.

Reply:
That won't work either. The main module is dotnet. I also cannot use modules list because Tracee is somewhere in the middle of it, probably due to load order.